tokio/src/net/windows/named_pipe.rs

```diff
@@ -2307,11 +2315,11 @@ impl ClientOptions {
     /// automatically when using this method.
     ///
     /// [`CreateFile`]: https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-createfilea
-    /// [`SECURITY_IDENTIFICATION`]: crate::winapi::um::winbase::SECURITY_IDENTIFICATION
+    /// [`SECURITY_IDENTIFICATION`]: crate::windows_sys::Win32::Storage::FileSystem::SECURITY_IDENTIFICATION
```

Comment:
@Darksonn any objection to just linking to the docs rather than going through these placeholder types. For example this would point to: 

https://docs.rs/windows-sys/*/windows_sys/Win32/Storage/FileSystem/constant.SECURITY_IDENTIFICATION.html

Reply:
Just linking directly is probably fine for doc-links. There are also the traits though, e.g. the `AsRawHandle` placeholder trait.